### PR TITLE
Add a devtool recipe for Google's gn

### PR DESCRIPTION
**Pull request by r1mikey**
Add a gn recipe, buildable as a native devtool.

Use this built gn when building Chromium.

### CODE REVIEW

**Comment by jaragunde** [comment]
Thanks for your contribution! Give me some time to try this out.

File: recipes-browser/chromium/chromium-browser-gn.inc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 include chromium.inc
 
 DESCRIPTION = "Chromium browser"
+DEPENDS += "gn-native ninja-native"
```
Comment (KurtSchluss) [inline recipes-browser/chromium/chromium-browser-gn.inc]: ninja-native is already set in chromium.inc:
https://github.com/Igalia/meta-browser/blob/ozone/wayland/20172906/recipes-browser/chromium/chromium.inc#L2

File: recipes-devtools/gn/gn.inc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+S ?= "${WORKDIR}/chromium-${PV}"
+
+do_compile () {
+  cd ${S}/tools/gn
+  ./bootstrap/bootstrap.py -s
+}
+
+do_install () {
+	install -d ${D}${bindir}
```
Comment (KurtSchluss) [inline recipes-devtools/gn/gn.inc]: The indentation for do_compile and do_install does not match the rest of your commit.

**Review by KurtSchluss — COMMENTED** [review]
This looks pretty got to me with just two small remarks.
I have tested this and it is working as expected.

**Comment by r1mikey** [comment]
Thanks for the review @KurtSchluss - I've addressed these comments.

**Comment by jaragunde** [comment]
Thank you so much for your contribution, and also to Kurt for the review. I've also tested it now.

Hopefully, this will be useful for somebody to update Chromium versions in the original meta-browser project.

**Pull request merged**